### ISSUE TITLE
Selected node label hard to read

### ISSUE THREAD

**Issue by maxkfranz** [cytoscape/enrichment-map-webapp#225]
**Describe the bug**
_A clear and concise description of what the bug is._

It's hard to read the selected node label when it's crowded.

**To Reproduce**
1. Expand a bubble.
2. Click a node in the bubble.
3. Can you read its label?  Usually, I can't unless I drag the node out.

**Expected behavior**
_A clear and concise description of what you expected to happen._

- The selected node should be on top.
- The selected node should be black and its label outline should be black.
- When expanding a bubble, it should apply a grid layout to the children so that they don't overlap.  We can fade out the other nodes, outside of the bubble, kind of like the Wine and Cheese app.

**Screenshots**

<img width="509" alt="Screenshot 2024-01-12 at 09 07 38" src="https://github.com/cytoscape/enrichment-map-webapp/assets/989043/88532f4a-aa32-49bd-813f-9ec43d3d4297">

**Comment by mikekucera** [comment]
We discussed using a grid layout when the cluster is expanded. We could also fade the rest of the network when a cluster is selected.

**Comment by chrtannus** [comment]
Indeed, the label is more legible when the selected node is on top and the label outline has the same dark color as the node border:

BEFORE:
  <img width="377" alt="blue-label-outline" src="https://github.com/cytoscape/enrichment-map-webapp/assets/989686/6ef7e52a-c0e1-4da1-bd7b-21bf29a9f7ca">
AFTER:
  <img width="374" alt="black-label-outline" src="https://github.com/cytoscape/enrichment-map-webapp/assets/989686/90888e33-55b5-4edb-949c-6986aa26590a">

Of course, this is only true if the adjacent/overlapping nodes are not selected as well.
